Sources/SwiftKuery/Table.swift

```diff
@@ -317,18 +283,18 @@ open class Table: Buildable {
     public func primaryKey(_ columns: Column...) -> Self {
         return primaryKey(columns)
     }
-    
+
     /**
-     Function to set a multiple `Column` instance, as a composite foreign key, in the `Table` instance referencing multiple column in another Table.
-     The function also validates the columns to ensure they belong to the table and do not conflict with the definition of a foreign key.
+     Function to set a multiple `Column` instance, as a composite foreign key, in the `Table` instance referencing multiple columns in another Table.
+     The function also validates the columns to ensure they belong to the table and do not conflict with the definition of an existing foreign key.
      ### Usage Example: ###
-     In this example, `Table` instances called personTable and employeeTable are created. A "personTable" foreign key is then set to be a composite of firstColumn and lastColumn, which reference firstName and surname in employeeTable.
+     In this example, `Table` instances called personTable and employeeTable are created. A composite primary key is created on "employeeTable". A "personTable" foreign key is then set to be a composite of firstColumn and lastColumn, which reference firstName and surname in employeeTable.
      ```swift
      public class EmployeeTable: Table {
-         let tableName = "employeeTable"
```

Comment:
Were you intending to change these from 4 to 3 spaces? Clogs up the diff a bit.

Reply:
The indentation in the two tables in the documentation was a space out, I'll change this and make them both consistent to 4 spaces.